src/components/controls/counter/Counter.js

```diff
@@ -19,16 +19,29 @@ const Counter = ({
         <p className="select-step"> ...or launch the cycle </p>
         {startButton}
       </div>
-      <div className="clock-container">
-        <div className="clock-balls">
-          <div className="clock-timer">
-            <span className="clock-counter">
-              {dayCount}
-            </span>
-            <span className="clock-dayer"> DAYS </span>
-          </div>
-        </div>
-      </div>
+      <svg
```

Comment:
You could factor this whole thing out to another React component called `Calendar` so that it's cleaner.

Reply:
I just did it before seeing your comment 😅